ghost/core/core/server/data/schema/schema.js

```diff
@@ -979,5 +979,20 @@ module.exports = {
         '@@UNIQUE_CONSTRAINTS@@': [
             ['email_id', 'member_id']
         ]
+    },
+    mentions: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        source: {type: 'string', maxlength: 2000, nullable: false},
+        source_title: {type: 'string', maxlength: 2000, nullable: true},
+        source_site_title: {type: 'string', maxlength: 2000, nullable: true},
+        source_excerpt: {type: 'string', maxlength: 2000, nullable: true},
+        source_author: {type: 'string', maxlength: 2000, nullable: true},
+        source_featured_image: {type: 'string', maxlength: 2000, nullable: true},
+        source_favicon: {type: 'string', maxlength: 2000, nullable: true},
+        target: {type: 'string', maxlength: 2000, nullable: false},
+        resource_id: {type: 'string', maxlength: 24, nullable: true},
+        resource_type: {type: 'string', maxlength: 50, nullable: true},
+        created_at: {type: 'dateTime', nullable: false},
+        payload: {type: 'text', maxlength: 65535, nullable: true}
```

Comment:
I'm still not convinced about the size of this 🤔 Seems massive but maybe we want that

Reply:
For now I followed the guidelines on top of [schema.js](https://github.com/TryGhost/Ghost/blob/45d65663f4af83cb4233a85d6555075298a34863/ghost/core/core/server/data/schema/schema.js#L7).

Reply:
But do we want to store it as text, or as a "large string" like 1000-2000 in size 🤔

Reply:
Let's just go with this for now 👍